instead of getting aports git, give them the actual commit sha in prepare
diff --git a/tools/get_git_commit.sh b/tools/get_git_commit.sh
index 4bf50d9..c003533 100755
--- a/tools/get_git_commit.sh
+++ b/tools/get_git_commit.sh
@@ -8,7 +8,7 @@ COMMIT=""
 if [ ! -d "./.git" ]; then
      echo "This is not a git directory and the git commit sha will remain undefined"
 else
-    SHA=$(git describe  --exclude '*' --abbrev=7 --always --dirty)
+    SHA="@@COMMIT@@"
     if [ -z "$SHA" ]; then
         echo "Undefined git commit version"
     else
